README.md

```diff
@@ -26,6 +28,22 @@ fn main() {
 }
 ```
 
+### Iterate over all MAC addresses
+
+```rust
+extern crate mac_address;
+
+use mac_address::MacAddresses;
+
+fn main() {
+    let addresses = MacAddresses::new().unwrap();
+
+    for address in addresses {
+        println!("{}", address)
+    }
+}
+```
+
 ## License
 
 `mac_address` is licensed under both MIT and Apache 2.0
```

Comment:
The README example is really just to give a quick example of the most common use-case, I'm not really sure its necessary to change it.